### ISSUE TITLE
Rename the package?

### ISSUE THREAD

**Issue by leouieda** [compgeolab/latex_variables#2]
Started using this for a paper and I think it would be useful to pick up development again. How about renaming to something without the underscore? 

The "grava" is up for grabs on PyPI. It means "record" in the imperative in Portuguese. I'm not 100% on that and would welcome suggestions of better names.

cc @santisoler

**Comment by santisoler** [comment]
I'm in for a rename! Similar to the Spanish word _grabar_ (the _to record_ verb), and _grava_ which means gravel, touching the geology side haha. I'm in for grava if you like it!

**Comment by leouieda** [comment]
I've been thinking how much this is actually needed. I managed to get by just fine with an f-string and siunitx. Without the unit parsing bits, this would mostly just declare the variables right?

**Comment by santisoler** [comment]
Yes. There's nothing too fancy here besides the units bits I think.

**Comment by leouieda** [comment]
Alright, let's keep this as is for now and then we can see if there's a need
